data/mods/gen9ssb/abilities.ts

```diff
@@ -883,6 +883,21 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		isPermanent: true,
 	},
 
+	// Sulo
+	protectionofthegelatin: {
+		shortDesc: "Magic Guard + Stamina",
```

Comment:
```suggestion
		shortDesc: "Immune to all non-attack damage; raises Defense by 1 when damaged by an attack.",
```
May as well just write it all out.